add retry logic and feature flag in microsegmentation UI

chandrasekhar1996 opened the pull request:
Signed-off-by: craman <chandrasekhar.raman@verizonmedia.com>

abvaidya commented:
Could you please add a little more details in comment as to what this PR is about?

chandrasekhar1996 commented:
Creation of microsegmentation policy involves creation of policy and assertion (if it doesn't exist) and then we need to fetch the assertionID of the newly created assertion to add assertionConditions.

I have added a retry logic for fetching the policy/assertion after creating it to account for read-after-write inconsistency. We try to fetch the policy twice with a 1 second sleep interval. 

I have also added a feature flag for the microsegmentation page. Currently we have a feature flag for the entire UI that we use to show/hide tabs. The new feature flag can be used to hide specific features in a page while under development.

pratikgote commented on ui/src/__tests__/components/microsegmentation/AddSegmentation.test.js:
```diff
@@ -25,7 +25,9 @@ describe('AddSegmentation', () => {
         let _csrf = 'csrf';
-
+        const pageFeatureFlag ={
+            "policyValidation": true
```
please replace with single quotes

pratikgote replied:
also, move this to consts file so it can be reused in line 74
